### PR TITLE
task 03 solved

### PR DESCRIPTION
**Pull request by arthur-borisenko**

### CODE REVIEW

File: week5/03-flaghi/task.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+                       "|__\\ ",
+                       "|    "]
+
+
+def main():
+    n = int(input())
+    result_lines = ["", "", "", ""]
+    for i in range(n):
+        for j in range(len(FLAG_LINES_TEMPLATE)):
+            if j == 1:
```
Comment (IgorNB) [inline week5/03-flaghi/task.py:12]: 1 - Magic number

File: week5/03-flaghi/task.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                result_lines[j] += FLAG_LINES_TEMPLATE[j].format(
```
Comment (IgorNB) [inline week5/03-flaghi/task.py:13]: result_lines[j] += FLAG_LINES_TEMPLATE[j] -- duplicated code

**Review by IgorNB — CHANGES_REQUESTED** [review]

File: week5/03-flaghi/task.py (hunk 2 above)
Comment (arthur-borisenko) [reply week5/03-flaghi/task.py:13]: done

File: week5/03-flaghi/task.py (hunk 1 above)
Comment (arthur-borisenko) [reply week5/03-flaghi/task.py:12]: done

**Review by IgorNB — APPROVED** [review]

**Pull request merged**